Formula/qt5.rb

```diff
@@ -199,6 +199,13 @@ def install
               /\n# pkgconfig\n(PKG_CONFIG_(SYSROOT_DIR|LIBDIR) = .*\n){2}\n/,
               "\n"
 
+    # Make `HOMEBREW_PREFIX/lib/qt5/plugins` an additional plug-in search path
+    # for Qt Designer to support formulae that provide Qt Designer plug-ins.
+    system "/usr/libexec/PlistBuddy",
+           "-c", "Add :LSEnvironment:QT_PLUGIN_PATH string \"#{HOMEBREW_PREFIX}/lib/qt5/plugins\"",
+           "#{bin}/Designer.app/Contents/Info.plist"
+    touch "#{bin}/Designer.app" # re-register edited bundle with Launch Services
```

Comment:
I guess my point here is that that is exactly what [the runtime options](http://doc.qt.io/qt-5/deployment-plugins.html) of `qt.conf` and `QT_PLUGIN_PATH` are meant to be used for.

Usually, for custom setting for an app bundle, I would use `qt.conf`.